libcidr-shopify.rb

```diff
@@ -0,0 +1,18 @@
+class LibcidrShopify < Formula
+  desc 'Libcidr is a library that provides utility functions to handle IP addresses and net blocks.'
+  homepage 'https://www.over-yonder.net/~fullermd/projects/libcidr'
+  url 'https://www.over-yonder.net/~fullermd/projects/libcidr/libcidr-1.2.3.tar.xz'
+  sha256 'afbe266a9839775a21091b0e44daaf890a46ea4c2d3f5126b3048d82b9bfbbc4'
+
+  patch do
+    url "https://gist.githubusercontent.com/fmejia97/ad72a1f2b5d4b77a2921f32d861850b8/raw/cd2d5d2c3737f585fc71226786ea82279bf36645/mac_os_libcidr_support.patch"
```

Comment:
I think you still need to update the patch url to the one in the commit here... https://raw.githubusercontent.com/Shopify/homebrew-shopify/3e72d2602b76e28036b1c8dded7f61569ba184a4/patch/mac_os_libcidr_support.patch